package.json

```diff
@@ -36,7 +36,7 @@
 	},
 	"homepage": "https://github.com/bitrise-io/bitrise-workflow-editor#readme",
 	"dependencies": {
-		"@bitrise/bitkit": "^9.0.0",
```

Comment:
Why shouldn't we allow to be updated to e. g. 9.0.1?

Reply:
Because there are breaking changes in the 9+ versions, didn't checked in which one, but if we want to update bitkit then we should  probably go through all the *.tsx files, there was a lot of issues.

Reply:
Because semantic versioning might sometimes be off, resulting in minor breaking changes within 9.x.x